### PR TITLE
Update mesh-api.yaml

### PR DESCRIPTION
**Pull request by mick-sch**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/mesh-api.yaml [hunk 1]
```diff
@@ -242,11 +256,9 @@ info:
     - send your business use case to [sa.servicedesk@nhs.net](mailto:sa.servicedesk@nhs.net)
     - upon receiving the SCAL (Supplier Conformance Assessment List) fill in and return to [sa.servicedesk@nhs.net](mailto:sa.servicedesk@nhs.net)
     - your integration will now be certificated by Solutions Assurance and ready to publish
-    - any other queries please contact us via our support mailbox; [ssd.nationalservicedesk@nhs.net](mailto:ssd.nationalservicedesk@nhs.net)
+    - for any other queries, contact our support mailbox [ssd.nationalservicedesk@nhs.net](mailto:ssd.nationalservicedesk@nhs.net)
 
-    ### Information
-    'Try this now' feature on the below endpoints are currently unavailable and awaiting removal.
-  contact:
+contact:
```
Comment (openfinch) [inline specification/mesh-api.yaml:261]: Check your indentation here.

**Review by openfinch — CHANGES_REQUESTED** [review]

**Review by mick-sch — COMMENTED** [review]
Fixed indent on contact

**Review by mick-sch — COMMENTED** [review]
Deleted contact: section

**Review by mick-sch — COMMENTED** [review]
restored contact: section with original indent

File: specification/mesh-api.yaml (hunk 1 above)
Comment (mick-sch) [reply specification/mesh-api.yaml]: Fixed - thanks.

**Review by openfinch — APPROVED** [review]

**Pull request merged**